pkg/types/ruletypes/api_params.go

```diff
@@ -143,15 +219,25 @@ func (r *PostableRule) processRuleDefaults() error {
 				Kind: BasicThresholdKind,
 				Spec: BasicRuleThresholds{{
 					Name:        thresholdName,
-					RuleUnit:    r.RuleCondition.CompositeQuery.Unit,
 					TargetUnit:  r.RuleCondition.TargetUnit,
 					TargetValue: r.RuleCondition.Target,
 					MatchType:   r.RuleCondition.MatchType,
 					CompareOp:   r.RuleCondition.CompareOp,
+					Channels:    r.PreferredChannels,
 				}},
 			}
 			r.RuleCondition.Thresholds = &thresholdData
 			r.Evaluation = &EvaluationEnvelope{RollingEvaluation, RollingWindow{EvalWindow: r.EvalWindow, Frequency: r.Frequency}}
+			r.NotificationSettings = &NotificationSettings{
+				Renotify: Renotify{
+					Enabled:          true,
+					ReNotifyInterval: Duration(4 * time.Hour),
+					AlertStates:      []model.AlertState{model.StateFiring},
+				},
+			}
+			if r.RuleCondition.AlertOnAbsent {
+				r.NotificationSettings.Renotify.AlertStates = append(r.NotificationSettings.Renotify.AlertStates, model.StateNoData)
+			}
```

Comment:
If you add it here by default, you need to make sure that they are set to `nil` in the `MarshalJSON` for `DefaultSchemaVersion`.